test_runner/batch_others/test_wal_acceptor_async.py

```diff
@@ -147,6 +147,10 @@ async def run_restarts_under_load(pg: Postgres, acceptors: List[Safekeeper], n_w
     period_time = 10
     iterations = 6
 
+    # Set timeout for this test at 15 minutes. It should be enough for
+    # test to complete and less than CircleCI's no_output_timeout.
```

Comment:
`It` --> `It plus a single iteration timeout` or something. Because we only check once per iteration.